users/services/views.py

```diff
@@ -77,3 +77,14 @@ def view(request, business_id):
     context["firstname"] = request.user.first_name
 
     return render(request, "services/view_business.html", context)
+
+
+@login_required
+def delete_business(request, business_id):
```

Comment:
Do you think we should introduce a confirmation before deleting the business?
We can add a new template `services/delete_business.html` which take the confirmation and deletes it later.

WDYT?